src/IQFeed.CSharpApiClient.Extensions/Lookup/Historical/Resample/TickMessageExtensions.cs

```diff
@@ -43,6 +43,7 @@ private static IEnumerable<HistoricalBar> ToHistoricalBarsAscending(this IEnumer
         {
             var intervalTicks = interval.Ticks;
             DateTime? nextTimestamp = null;
+            DateTime? currentBarDate = null;
```

Comment:
very minor.... can we use currentDate instead? like we dont use nextBarTimestamp